modules/browser/BrowserCapabilities.js

```diff
@@ -241,6 +255,39 @@ export default class BrowserCapabilities extends BrowserDetection {
         return this.isFirefox();
     }
 
+    /**
+     * Checks if the browser uses unified plan by default.
+     *
+     * @returns {boolean}
+     */
+    isUnifiedPlanDefault() {
+        if (this.isFirefox()) {
+            return true;
+        }
+
+        if (this.supportsSdpSemantics()) {
+            // FIXME: Find a more elegant solution without creating a dummy
+            // connection, the code doesn't go this branch for now as Chrome's
+            // semantics is set explicitly in RTC.js
+            const pc = new RTCPeerConnection();
+            const semantics = pc.getConfiguration().sdpSemantics;
+
+            pc.close();
+
+            return semantics === 'unified-plan';
+        }
+
+        if (this.isSafariWithVP8()) {
```

Comment:
Same as above, semantically using VP8 support as the condition here is very counter intuitive. Also, This check should be part of the `usesUnifiedPlan` function.

Reply:
I've made some fixes for sdp-interop, Safari works in both modes with those changes.
https://github.com/jitsi/sdp-interop/pull/28
I think it's more reasonable to use `usesUnifiedPlan`.